alerts/kube_apiserver.libsonnet

```diff
@@ -75,7 +75,7 @@ local utils = import '../lib/utils.libsonnet';
           {
             alert: 'KubeAggregatedAPIErrors',
             expr: |||
-              sum by(name, namespace, %(clusterLabel)s)(increase(aggregator_unavailable_apiservice_total[10m])) > 4
+              max by(name, namespace, %(clusterLabel)s)(increase(aggregator_unavailable_apiservice_total[10m])) > 4
```

Comment:
Seems like this changes the intention of the alert slightly. Agree that HA scenario should be de-duped, but the `Aggregated` part of the alert name implies that we want to look at the total number of errors across all services, rather than any individual service.

So may you need to combine the aggregation and de-duplication with: `sum by (x) (max by (y) (...))`?